test/test_helper.exs

```diff
@@ -75,6 +75,27 @@ defmodule SSEClient do
 
 end
 
+defmodule ControlClient do
```

Comment:
Do we want to expose a client like this as part of the project's available tools?

Reply:
I don't think there's any real benefit to that, because if you need such a thing in Elixir you can probably just use the `SSEServer` API. I also don't really want to make the HTTP client a non-test dependency.

Reply:
Right.